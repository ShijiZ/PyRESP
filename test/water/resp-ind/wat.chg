All values are reported in atomic units
%FLAG TITLE
 molecule 1: water

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1  -9.9821230E-33   0.0000000E+00   2.3138460E-01
   2  -2.6101230E-33   1.4941870E+00  -9.2553830E-01
   3  -1.8298510E-16  -1.4941870E+00  -9.2553830E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          8          0    -8.4263311E-01
   2          1          0     4.2131655E-01
   3          1          2     4.2131655E-01

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   1.6996824E-17   0.0000000E+00   2.1492477E-01
   2   2.8759880E-18  -2.2064212E-02   5.3450789E-02
   3   5.5780741E-18   2.2064212E-02   5.3450789E-02

