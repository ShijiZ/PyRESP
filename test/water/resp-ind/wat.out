
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 resp-ind for water
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: water
 Total charge (ich):  0
 Number of centers:  3
    1    8    0
    2    1    0
    3    1    2

 -----------------------------
 Atom Ivary Information
 -----------------------------
    8    0
    1    0
    1    2

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =    3
 Weight factor on initial charge restraints=  0.000500
 There are  1 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =     3
 total number of esp points =   295

 center       X               Y               Z
    1  -9.9821230E-33   0.0000000E+00   2.3138460E-01
    2  -2.6101230E-33   1.4941870E+00  -9.2553830E-01
    3  -1.8298510E-16  -1.4941870E+00  -9.2553830E-01


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 values will be set to 0

 Number of unique UNfrozen charge centers =     2

 Non-linear optimization requested.
  qchnge =    3.4426216756E-01
  qchnge =    2.5737849102E-04
  qchnge =    1.2580835053E-06
  qchnge =    6.1541026025E-09

 Convergence in    3 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: water

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         8        0.000000     -0.842633      0       0.000589
    2         1        0.000000      0.421317      0       0.000000
    3         1        0.000000      0.421317      2       0.000000

 Sum over the calculated charges:     -0.000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          8.5437555E-02
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   6.7775602E-04
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.5157434E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         8.9066069E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        1.65986   -0.00000    0.00000   -1.65986

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   -1.04361
        Y    0.00000    1.30940
        Z    0.00000    0.00000   -0.26579

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X    1.30940
        Y    0.00000   -0.26579
        Z    0.00000    0.00000   -1.04361